slash/react/src/Layout/Footer/Footer.tsx

```diff
@@ -8,6 +8,7 @@ type FooterProps = {
   alt?: string;
   icon?: string;
   className?: string;
+  version: string;
```

Comment:
mettre la props en optionnel, sinon celui produira un breaking change
```suggestion
  version?: string;
```

Reply:
c'est fait :)